### PR TITLE
docs: add MAINTAINERS.md file

### LINKED ISSUE #80: Add different repository files
We need the following files in our repo:

* CONTRIBUTING.md: to guide new contributors
* CODE_OF_CONDUCT.md: how we want contributors to behave and how we enforce that or respond to violations of the code
* Maintainers file: list of maintainers, definition, processes related with maintainers, etc.

### PR DESCRIPTION
**Pull request by rh-hemartin**
Closes  #80

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+The current Maintainers Group for the KubeArchive Project consists of:
+
+* [Andrea Fernandez](https://github.com/mafh314), [Red Hat](https://www.redhat.com)
```
Comment (maruiz93) [inline MAINTAINERS.md:3]: Maybe Andrea Fernández (with the accent) ?
@mafh314 what do you think?
```suggestion
* [Andrea Fernández](https://github.com/mafh314), [Red Hat](https://www.redhat.com)
```

File: MAINTAINERS.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+* [Chris Beer](https://github.com/beerparty), [Red Hat](https://www.redhat.com)
+* [Greg Allen](https://github.com/ggallen), [Red Hat](https://www.redhat.com)
+* [Hector Martinez](https://github.com/rh-hemartin), [Red Hat](https://www.redhat.com)
```
Comment (maruiz93) [inline MAINTAINERS.md:6]: As we are going with accents maybe you prefer having the accents?
```suggestion
* [Héctor Martínez](https://github.com/rh-hemartin), [Red Hat](https://www.redhat.com)
```

Comment (rh-hemartin) [reply MAINTAINERS.md:6]: I don't use my accents in a professional setting, as there is a lot of programs that do not support them.

File: MAINTAINERS.md (hunk 1 above)
Comment (mafh314) [reply MAINTAINERS.md:3]: yes, please :)

Comment (rh-hemartin) [reply MAINTAINERS.md]: Added!

**Review by beerparty — APPROVED** [review]
LGTM

**Pull request merged**